gef.py

```diff
@@ -9943,6 +9946,13 @@ def load_plugins_from_directory(plugin_directory: pathlib.Path):
         dbg(f"Loading extra plugins from directory={directory}")
         return load_plugins_from_directory(directory)
 
+
+    def load_virtualenv(self, new_path: Optional[pathlib.Path] = None):
+        path = new_path or gef.config["gef.virtualenv_path"]
+        if path:
+            activate_script_path = pathlib.Path(path)/"bin"/"activate_this.py"
+            exec(open(activate_script_path).read(), {'__file__': activate_script_path})
```

Comment:
Avoid cascading by returning first on error
```suggestion
        if not path:
            return
        activate_script_path = pathlib.Path(path)/"bin"/"activate_this.py"
        exec(open(activate_script_path).read(), {'__file__': activate_script_path})
```